package.json

```diff
@@ -149,7 +149,7 @@
     "less": "2.7.1",
     "less-loader": "2.2.3",
     "lodash": "3.10.1",
-    "marked": "0.3.6",
+    "markdown-it": "8.3.2",
```

Comment:
Its an easy configuration change to enable. I am +1 on enabling. What do you think?